docs/cicd.md

```diff
@@ -0,0 +1,214 @@
+
+### iOS Variables
+
+| Variable                          | Description                                                                                                                | Where do I get it from? | Example                                                                    | Reference                                                                                   |
+| --------------------------------- | -------------------------------------------------------------------------------------------------------------------------- | ----------------------- | -------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| APP_STORE_CONNECT_API_KEY_ID      | Key ID for App Store Connect API                                                                                           | Password Manager        | D83848D23                                                                  | [app_store_connect_api_key](https://docs.fastlane.tools/actions/app_store_connect_api_key/) |
+| APP_STORE_CONNECT_API_ISSUER_ID   | Issuer ID for App Store Connect API                                                                                        | Password Manager        | 227b0bbf-ada8-458c-9d62-3d8022b7d07f                                       | [app_store_connect_api_key](https://docs.fastlane.tools/actions/app_store_connect_api_key/) |
+| APP_STORE_CONNECT_API_KEY_CONTENT | Key content for App Store Connect API                                                                                      | Password Manager        | -----BEGIN EC PRIVATE KEY-----\nfewfawefawfe\n-----END EC PRIVATE KEY----- | [app_store_connect_api_key](https://docs.fastlane.tools/actions/app_store_connect_api_key/) |
+| MATCH_PASSWORD                    | Password for accessing the certificates for the iOS app using [Fastlane Match](https://docs.fastlane.tools/actions/match/) | Password Manager        | 123456                                                                     | [Using a Git Repo](https://docs.fastlane.tools/actions/match/#git-repo-encryption-password) |
+
+
+## Hints and Quirks
+
+### CPU count aka. \$TOTAL_CPUS
+
+There is no obvious way for an application to know how many cores it has available in a CircleCI docker container. The usual ways of getting the CPU count reports the CPU count of the host. This causes out-of-memory issues as the host has a lot of cores.
+Therefore, all tools must set the worker limit to `$TOTAL_CPUS`. Set this variable in the `.circleci/config.yml`.
+
+## Troubleshooting
+
+This section lists some commands that may help you to find and solve issues in a new deployment on staging or production server.
```

Comment:
Nice thanks for adding these instructions. Please move them to a separate file as this is not CICD releated.